### DISCUSSION TITLE
Where do I get the EFI files?

### DISCUSSION THREAD

**Discussion by RishonDev** [ventoy/Ventoy#2069] [Q&A]
Hi! I want to integrate ventoy support into my [app](https://github.com/WaveUSB-Beta). Where can I find the EFI files?

**Comment by uuaws2** [comment]
I honestly dont know if you can get them. Maybe you can rip some stuff out of the windows xp or other efi based operation files?...

**Reply by RishonDev** [reply]
No like the ventoy EFI file.

**Reply by uuaws2** [reply]
ohhhh your talking about the efi files for ventoy , yeah i think i got from a website its url was www.efi.ventoy.com but i think they are closed.
